### PR TITLE
Add Psychology Statistics course

### PR DESCRIPTION
**Pull request by krcarriere**
Please see course description in .rmd for details on what this course covers.

### CODE REVIEW

File: Psychology_Statistics.Rmd [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+### Installation
+
+```r
+swirl::install_course("Psychology Statistics")
+```
+
+#### Manual Installation
+
+1. Download [this](http://swirlstats.com/scn/R_Programming.swc) file.
```
Comment (seankross) [inline Psychology_Statistics.Rmd:23]: Note to self: change this line to point to appropriate `.swc` file.

**Comment by WilCrofter** [comment]
Nice! Glad to see things going so well. Good job, Sean & co.

**Comment by seankross** [comment]
Hey @WilCrofter! I hope you're doing well!

**Comment by WilCrofter** [comment]
Hey @seankross. Yep. Doing well. Focused on Julia rather than R these days. Currently, at Sasha's behest, building an experimental cloud computing environment at home. Following swirl, you, and Nick intermittently but with great interest. Cheers--Bill.

**Comment by seankross** [comment]
Thank you @krcarriere!

**Pull request merged**